new-frontiers-14/frontier-station-14
# CrewMonitoringNavMapControl hotfix

## About the PR



Fixes CrewMonitoringNavMapControl.

## Why / Balance


No XAML, no need for RobustXamlLoader.  Goofed in the upstream merge.

## How to test


Open the crew monitoring console.  It works, check the coords, they should make sense.

## Media


![image](https://github.com/user-attachments/assets/0c14aa45-e18d-44ae-ba6f-5d31dd0c246f)

## Requirements



## Breaking changes


**Changelog**


No CL, not yet live.
